### PR TITLE
Add Citus CDC guide

### LINKED ISSUE #773: Document ingesting Citus distributed table using citus-cdc connector
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/8988

### Which part(s) of the docs might be affected or should be updated? And how?

Add a new page in Ecosystem - Sources

Update the description accordingly: https://www.risingwave.dev/docs/upcoming/data-ingestion/#change-data-capture-cdc

### Reference

See the description in code PR

### PR DESCRIPTION
**Pull request by emile-00**
## Info
- **Description**: 
Add Citus CDC guide, connector node instructions are outdated to be consistent with the other guides. Will be updated once the other PR is approved.

- **Preview**: 
[ Paste the preview link to the edited page here. ]

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/8988

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/773

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/guides/ingest-from-citus-cdc.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+);
+```
+
+### WITH parameters
+
+Unless specified otherwise, the fields listed are required. 
+
+|Field|Notes|
+|---|---|
+|hostname| Hostname of the database. |
```
Comment (StrikeW) [inline docs/guides/ingest-from-citus-cdc.md]: the hostname of the coordinator node

File: docs/guides/ingest-from-citus-cdc.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+|port| Port number of the database.|
```
Comment (StrikeW) [inline docs/guides/ingest-from-citus-cdc.md]: port of the coordinator node

File: docs/guides/ingest-from-citus-cdc.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+There are a few limitations when ingesting CDC data from Citus in RisingWave.
+
+- A Postgres `superuser` role is required.
+- Only [distributed tables](https://docs.citusdata.com/en/v10.2/get_started/concepts.html#table-types) are supported.
+- Newly added worker nodes are not detected.
+
+## Enable the connector node in RisingWave
+
+The native PostgreSQL CDC connector is implemented by the connector node in RisingWave. The connector node handles the connections with upstream and downstream systems. You can use the docker-compose configuration of the latest RisingWave demo, in which the connector node is enabled by default. To learn about how to start RisingWave with this configuration, see [Docker Compose](../deploy/risingwave-docker-compose.md).
```
Comment (StrikeW) [inline docs/guides/ingest-from-citus-cdc.md]: For users who are willing to try the open-source project, they can build and launch the connector node via `risedev`
1. `./risedev configure` to enable Build RisingWave Connector 
2. Edit the profile in  `risdev.yml` to add `use: connector-node`

We should notice users for that in the doc. cc @CharlieSYH

**Review by StrikeW — APPROVED** [review]
Rest LGTM

File: docs/guides/ingest-from-citus-cdc.md (hunk 3 above)
Comment (emile-00) [reply docs/guides/ingest-from-citus-cdc.md]: Thanks for pointing it out! The goal is to have the connector node instructions and any guide that mentions the connector node be updated with [this PR](https://github.com/risingwavelabs/risingwave-docs/pull/633).

File: docs/guides/ingest-from-citus-cdc.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
+---
+ id: ingest-from-citus-cdc
+ title: Ingest data from Citus CDC
+ description: Describes how to ingest data from Citus CDC.
+ slug: /ingest-from-citus-cdc
+---
+
+Change Data Capture (CDC) refers to the process of identifying and capturing data changes in a database, then delivering the changes to a downstream service in real time.
+
+Citus database is an extension to Postgres that transforms Postgres into a distributed database. For more details, see [Citus data](https://www.citusdata.com).
```
Comment (hengm3467) [inline docs/guides/ingest-from-citus-cdc.md]: @emile-00 A minor one. Let's use "PostgreSQL" all the time.

File: docs/guides/ingest-from-citus-cdc.md (hunk 3 above)
Comment (hengm3467) [inline docs/guides/ingest-from-citus-cdc.md]: @emile-00 Let's ensure that PR is accurate and publish it, and then reference it from here (updating the instructions as we should not use demo repo as the preferred way to enable the node)

**Comment by hengm3467** [comment]
Otherwise looks good to me.

**Review by bernscode — APPROVED** [review]

File: docs/guides/ingest-from-citus-cdc.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+The native PostgreSQL CDC connector is implemented by the connector node in RisingWave. The connector node handles the connections with upstream and downstream systems. You can use the docker-compose configuration of the latest RisingWave demo, in which the connector node is enabled by default. To learn about how to start RisingWave with this configuration, see [Docker Compose](../deploy/risingwave-docker-compose.md). 
+
+## Create a table in RisingWave using the native CDC connector
+
+### Syntax
+
+```sql
+CREATE TABLE [ IF NOT EXISTS ] source_name (
+    column_name data_type PRIMARY KEY , ...
```
Comment (CharlieSYH) [inline docs/guides/ingest-from-citus-cdc.md:59]:
```suggestion
    column_name data_type [ PRIMARY KEY ], ...
```

File: docs/guides/ingest-from-citus-cdc.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    PRIMARY KEY ( column_name, ... )
```
Comment (CharlieSYH) [inline docs/guides/ingest-from-citus-cdc.md]:
```suggestion
    [ PRIMARY KEY ( column_name, ... ) ]
```

**Review by CharlieSYH — APPROVED** [review]

File: docs/guides/ingest-from-citus-cdc.md (hunk 5 above)
Comment (emile-00) [reply docs/guides/ingest-from-citus-cdc.md:59]: If I recall correctly, a primary key is required for tables created with Citus CDC as the source.

**Comment by StrikeW** [comment]
Hi @emile-00 , please add the supported version of Citus to the doc: Citus v10.2, since we only tested v10.2 before.

**Pull request merged**